skimage/feature/_hog.py

```diff
@@ -230,12 +230,11 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
     normalized_blocks = np.zeros((n_blocksy, n_blocksx,
                                   by, bx, orientations))
 
-    eps = 1e-8
```

Comment:
ah I see, you prefer for `eps` to be set as a default parameter of `_hog_normalize_block`?  I'll add the default value back and not specify a value for `eps` in the method call below.